docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+# How To Contribute
+
+## Some Ways to Contribute
+- [Getting started with L3AF](#getting-started-with-l3af)
+- [Report potential bugs](#report-potential-bugs)
+- [New Features or product enhancements](#new-features-or-product-enhancements)
+- [Submitting a patch that fixes a bug](#submitting-a-patch-that-fixes-a-bug)
+- [Contribute code](#coding-style)
+- [Guidelines for pull requests](#guidelines-for-pull-requests)
+- [Improve our guides and documentation](#improve-our-guides-and-documentation)
+- [Increase our test coverage](#increase-our-test-coverage)
+
+### Getting started with L3AF
+
+See [L3AF](https://wiki.lfnetworking.org/display/L3AF/Getting+Started+with+L3AF)
+
+### Report potential bugs
+
+First, **ensure the bug was not already reported** by searching on GitHub under
+[Issues](https://github.com/l3af-project/l3afd/issues).
+
+If you did not find a related bug, you can help us by
+[submitting a GitHub Issue](https://github.com/l3af-project/l3afd/issues/new).
+A good bug report provides a detailed description of the issue and step-by-step instructions
+for reliably reproducing the issue.
+
+We will aim to triage issues in [weekly TSC meetings](https://wiki.lfnetworking.org/display/L3AF/Community+Meetings).
+In case we are unable to repro the issue, we will request more information from you. There will be a waiting period of
+2 weeks   for the requested information and if there is no response, the issue will be closed. If this happens,
+please reopen the issue if you do get a repro and provide the requested information.
+
+If you found a security issue, please do not open a GitHub Issue, and instead [email](security@lists.l3af.io) it in detail.
+
+### New Features or product enhancements
+
+You can request or implement a new feature by [submitting a GitHub Issue](https://github.com/l3af-project/l3afd/issues/new).
+and communicate your proposal so that the L3AF community can review and provide feedback. Getting
+early feedback will help ensure your implementation work is accepted by the community.
+This will also allow us to better coordinate our efforts and minimize duplicated effort.
+
+### Submitting a patch that fixes a bug
+
+Fork the repo and make your changes. Then open a new GitHub pull request with the patch.
+
+* Ensure the PR description clearly describes the problem and solution. Include the relevant issue number
+  if applicable.
+
+### Guidelines for pull requests
```

Comment:
```suggestion
#### Guidelines for pull requests
```
Indent a level since not in the TOC at top of page

Reply:
It is present in the TOC.